chia/cmds/coin_funcs.py

```diff
@@ -131,7 +132,7 @@ async def async_combine(args: Dict[str, Any], wallet_client: WalletRpcClient, fi
     largest = bool(args["largest"])
     if target_coin_amount != 0:  # if we have a set target, just use standard coin selection.
         removals: List[Coin] = await wallet_client.select_coins(
-            amount=target_coin_amount + fee,
+            amount=target_coin_amount + fee if is_xch else target_coin_amount,
```

Comment:
this doesn't look right. you add `target_coin_amount` twice for non-standard wallet coins?

Reply:
what happened here? it still doesn't look right? did you change this in another commit?